src/js/stores/library.js

```diff
@@ -504,6 +522,15 @@ define(function (require, exports, module) {
         },
         
         /**
+         * Ture if DS is in placing graphic mode.
+         * 
+         * @return {boolean}
+         */
+        isPlacingGraphic: function () {
```

Comment:
side note, for future, I think we should strive to use the semi-standard `getState()` to provide simple store properties instead of individual getter methods like this.